src/main/java/org/fcrepo/migration/PicocliMigrator.java

```diff
@@ -105,6 +105,10 @@ static F3SourceTypes toType(final String v) {
             description = "Type of OCFL objects to migrate to. Choices: FEDORA_OCFL | PLAIN_OCFL")
     private MigrationType migrationType;
 
+    @Option(names = {"--id-prefix"}, defaultValue = "info:fedora/", showDefaultValue = ALWAYS, order = 20,
+            description = "Prefix to add to PIDs for OCFL object IDs - defaults to info:fedora/, like Fedora3")
```

Comment:
Can you add a note here that this should NOT be used for FEDORA_OCFL migrations?

Reply:
updated with a note & check that idPrefix stays as default for FEDORA_OCFL migrations